src/Select/__snapshots__/Select.test.js.snap

```diff
@@ -269,22 +269,23 @@ exports[`<Select /> create Select component 7`] = `
         onKeyPress={[Function]}
         role="button"
         tabIndex={0}
-        uvalidationState={
-          Object {
-            "state": "success",
-            "text": "Test validation state",
-          }
-        }
       >
         <div
-          className="fd-select__control"
+          className="fd-select__control is-success"
```

Comment:
This snapshot is changed because the previous test misspelled `validationState` as `uvalidationState`